ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/codes/RespConstantsV7.java

```diff
@@ -20,7 +20,40 @@
 
 public final class RespConstantsV7 {
 
+    public static final String SVAR_JA_TEXT = "Ja";
```

Comment:
This is a general comment. To follow the naming-convention.. shouldn't we remove the "number" in the end of TEXT_ID and DESCRIPTION_ID?

Reply:
Yes, I agree. Done.

Reply:
I think you changed even the SVAR_ID and DELSVAR ID now, which was there from before. I'm fine with that, but historically we haven't modified these :-)